controller/linodemachine_controller.go

```diff
@@ -294,7 +294,7 @@ func (r *LinodeMachineReconciler) reconcileCreate(
 	logger logr.Logger,
 	machineScope *scope.MachineScope,
 	clusterScope *scope.ClusterScope,
-) error {
+) (*linodego.Instance, error) {
```

Comment:
@AshleyDumaine there is a defer function which updates the status based on the linode instance status, so in this case we need the instance itself.